src/Microsoft.DotNet.Build.Tasks/PackageFiles/resources.targets

```diff
@@ -20,20 +20,25 @@
           GenerateResourcesSource;
           $(CompileDependsOn);
       </CompileDependsOn>
+      <ResourcesFolderPath Condition="'$(BuildingUAPVertical)' == 'true'">$(RuntimePath)resw</ResourcesFolderPath>
```

Comment:
I tried adding it to Build.Common.props but by the time resources.targets is called it is not defined yet.